runtime/str_test.go

```diff
@@ -306,6 +306,11 @@ func TestStrMethods(t *testing.T) {
 		{"capitalize", wrapArgs("вол"), NewStr("вол").ToObject(), nil},
 		{"capitalize", wrapArgs("foobar", 123), nil, mustCreateException(TypeErrorType, "'capitalize' of 'str' requires 1 arguments")},
 		{"capitalize", wrapArgs("ВОЛ"), NewStr("ВОЛ").ToObject(), nil},
+		{"center", wrapArgs("foobar", 10, "#"), NewStr("##foobar##").ToObject(), nil},
```

Comment:
How about a test case where the number of characters to pad are odd. Which side should the extra character go on?